### ISSUE TITLE
Plugin refresh button doesn't upsdate the list of plugins below

### ISSUE THREAD

**Issue by yurivict** [falkTX/Carla#832]
I started Carla, while having no LV2 plugins installed.
I installed many plugins, hit the "Refresh" button in carla.
Judging from its progress bar, it did find them, but the list below stayed the same.
I had to restart Carla to get it updated.

Carla-1.9.13 on FreeBSD 11.2

**Comment by SpotlightKid** [comment]
Yes, that has been my observation as well, that Carla only sees newly installed LV2 plugins after a restart.
